components/managedidentity/locals.tf

```diff
@@ -1,8 +1,8 @@
 locals {
   mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env
-  cosmosdb_name  = contains(["sandbox", "perftest", "dev", "sbox"], var.env) ? "sandbox-pipeline-metrics" : "pipeline-metrics"
-  cosmosdb_rg    = contains(["sandbox", "perftest", "dev", "sbox"], var.env) ? "pipelinemetrics-database-sandbox" : "pipelinemetrics-database-prod"
-  cosmosdb_env   = contains(["sbox", "dev"], var.env) ? "sandbox" : "prod"
+  cosmosdb_name = contains(["sbox", "dev", "demo", "ithc", "ptlsbox", "test"], var.env) ? "sandbox-pipeline-metrics" : "pipeline-metrics"
+  cosmosdb_rg   = contains(["sbox", "dev", "demo", "ithc", "ptlsbox", "test"], var.env) ? "pipelinemetrics-database-sandbox" : "pipelinemetrics-database-prod"
+  cosmosdb_env  = contains(["sbox", "dev", "demo", "ithc", "ptlsbox", "test"], var.env) ? "sandbox" : "prod"
 
```

Comment:
Haven't checked in detail, but does this mean helm-base-versions job is reporting to the sandbox-pipeline-metrics DB in all these envs?

And only sending aat&prod data to pipeline-metrics DB?